n2survey/plot/simple_comparison.py

```diff
@@ -0,0 +1,193 @@
+from typing import Dict, Optional, Union
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+__all__ = ["simple_comparison_plot"]
+
+
+def get_percentages(array, threshold=0, answer_supress=False, totalbar=False):
+    """
+    This function calculates the total number of combinations from the given
+    array of answer combinations.
+    After that it adds the totalbar, if wanted, then it calculates a dictionary
+    with the percentages for each combination+the totalbar if wanted
+    """
+    # get possible combinations (combi) and count how often they occure (no_of)
+    (combi, no_of) = np.unique(array.astype(str), return_counts=True, axis=0)
+    # if a totalbar array is calculated in the plot function survey.py
+    # it will be appended to the beginning of combi and no_of
+    if totalbar:
+        total_combi = np.full((totalbar[0].shape[0], 1), "Total")
+        total_combi = np.append(
+            total_combi, np.reshape(totalbar[0], newshape=(-1, 1)), axis=1
+        )
+        combi = np.append(total_combi, combi, axis=0)
+        no_of = np.append(totalbar[1], no_of)
+    # Total values are converted to percentages and packed into a dictionary,
+    # which is less confusing then pandas dataframe
+    percentage = {}
+    for answer in combi[:, 0]:
+        answer_rows = np.where(combi[:, 0] == answer)
+        percentage[answer] = np.append(
+            np.reshape(combi[answer_rows, 1], newshape=(-1, 1)),
+            np.reshape(
+                np.round(
+                    no_of[answer_rows] / sum(no_of[answer_rows]) * 100, decimals=1
+                ),
+                newshape=(-1, 1),
+            ),
+            axis=1,
+        )
+    return percentage
+
+
+def form_x_and_y(df, totalbar=False, answer_supress=False):
+    """
+    This Function splits up the dataframe given to it to x and y components
+    it applies the get_percentages function to calculate Percentages from
+    the compared with Question.
+    It also filters out the answers specified by answer_supress.
+    If additional answers should be compared this function also splits them up
+    for the get_percentages function which only can handle one array, not
+    a list of arrays.
+    """
+    PERCENTAGES = []
+    for entry in df:
+        PERCENTAGE = get_percentages(entry, totalbar=totalbar)
+        totalbar = False
```

Comment:
This one is important since the function has the option to compare more then one question with the "compare_with" variable I added the option to add bars from other questions to the comparison (add_questions=['A1','A2'...]). 
If this is not set  it will calculate and add the same totalbar of the compare_with for every question in add_questions.